Revert "storage: build SSTs from KV_BATCH snapshot"

tbg opened the pull request:
This reverts commit b320ff5c5a3338e2207496c88228f781e4b711b1.

In the above commit, we are starting both to ingest multiple SSTs at the
same time, and additionally these SSTs contain range deletion
tombstones. Both are firsts, and it turns out that there are some kinks
to work out. The commit causes quite a number of failures, so reduce
churn while we do so.

See https://github.com/cockroachdb/cockroach/issues/39604.

Release note: None

tbg committed:
Revert "storage: build SSTs from KV_BATCH snapshot"

This reverts commit b320ff5c5a3338e2207496c88228f781e4b711b1.

In the above commit, we are starting both to ingest multiple SSTs at the
same time, and additionally these SSTs contain range deletion
tombstones. Both are firsts, and it turns out that there are some kinks
to work out. The commit causes quite a number of failures, so reduce
churn while we do so.

See https://github.com/cockroachdb/cockroach/issues/39604.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/39619)
<!-- Reviewable:end -->

tbg committed:
storage: pacify staticcheck

The method has become unused with the preceding reverted commit.

Release note: None

tbg commented:
bors r=nvanbenschoten
